src/display/lv_display.c

```diff
@@ -1031,6 +1031,35 @@ void lv_display_rotate_area(lv_display_t * disp, lv_area_t * area)
     }
 }
 
+uint32_t lv_display_get_draw_buf_size(lv_display_t * disp)
+{
+    if(!disp) disp = lv_display_get_default();
+    if(!disp) return 0;
+
+    if(disp->buf_1) {
+        return disp->buf_1->data_size;
+    }
+    return 0;
+}
+
+uint32_t lv_display_get_invalidated_draw_buf_size(lv_display_t * disp, uint32_t width, uint32_t height)
+{
+    if(!disp) disp = lv_display_get_default();
+    if(!disp) return 0;
+
+    if(disp->render_mode == LV_DISPLAY_RENDER_MODE_FULL) {
+        return disp->buf_act->data_size;
```

Comment:
Thinking about it - the user might allocate a larger buffer then actually needed. So instead something like this would make sense, right?

```
        width = lv_display_get_horizontal_resolution(disp);
        height = lv_display_get_vertical_resolution(disp);
```

Reply:
Good idea! Yes, in case of FULL mode it makes perfect sense. 

However we need should use `stride` instead of `width` here too.